lib/tasks/report_inconsistent_facet_values.rake

```diff
@@ -0,0 +1,57 @@
+require "gds_api/rummager"
+require "json"
+
+class EmptyFacetValueError < StandardError
+end
+
+desc "Check for and report any non-expanded facet values"
+task :report_inconsistent_facet_values do
+  # List of facets to check
+  facets = %w{
```

Comment:
Perhaps we should check all the allowed facets that use the expansion mechanism:

https://github.com/alphagov/rummager/blob/master/lib/search/registries.rb#L11-L26